### DISCUSSION TITLE
Does Phase4 Support Wildcard Document Type ID

### DISCUSSION THREAD

**Discussion by jianweng11** [phax/phase4#278] [Q&A] [answered]
Dear Experts,

Does Phase4 support wildcard document type id? We hit "The value of 'DocTypeID' may not be null!" and cannot replace the wildcard with others value.

DocumentIdentifier: urn:oasis:names:specification:ubl:schema:xsd:Invoice-2::Invoice##urn:peppol:pint:billing-1*::2.1
ProcessIdentifier: urn:peppol:bis:billing

Thanks.

**Comment by phax** [comment, marked as answer]
Hi @jianweng11,
yes, the wildcard document type is supported.
Please note, that the `*` is only required for SMP lookups.
As soon as your document type ID uses `peppol-doctype-wildcard` as the scheme ID, a wildcard lookup is performed internally and the '*' is added automatically for the lookup only.
